src/components/QuickOpenModal.js

```diff
@@ -320,8 +320,13 @@ export class QuickOpenModal extends Component<Props, State> {
 
   /* eslint-disable react/no-danger */
   renderHighlight = (candidateString: string, query: string, name: string) => {
-    const html = fuzzyAldrin.wrap(candidateString, query);
-
+    const options = {
+      wrap: {
+        tagOpen: '<mark class="highlight">',
```

Comment:
At face value I like this change; two questions though:

1.  Do we need to update a CSS  style in our codebase for this?
2.  `<mark>` seems like the correct semantic choice, which is why I love this change, but any idea why they wouldn't use `<mark>` by default?  Any clue why `<strong>`?

Reply:
ad 1. the class 'highlight' is not tag dependent, however replacing 'strong' with 'mark' makes the items background yellow.

ad 2. The connected issue provides all information and links. In brief: The author was not aware it could be an issue and is considering it because we made him aware that it might. As far as I understood it he wants to bundle this change with some others so it might take some more time.
Speaking to Jason there was the notion to include this fix in favor of waiting for a native fix that may or may not come soon as it will fix the issue now and should have no negative impact in future where we might be able to get rid of the configuration option altogether.

Reply:
hmm, so I'm not a fan of the yellow background. if the markup makes it clearer, we can just remove the background via css